packages/smooth_app/lib/pages/preferences/user_preferences_page.dart

```diff
@@ -43,115 +43,147 @@ class _UserPreferencesPageState extends State<UserPreferencesPage> {
     final List<Widget> children = <Widget>[];
+    final bool addDividers;
+
     if (widget.type == null) {
-      children.add(UserPreferencesListTile(
-        title: Text(
-          appLocalizations.user_list_all_title,
-          style: themeData.textTheme.headline2,
-        ),
-        onTap: () async => Navigator.push<void>(
-          context,
-          MaterialPageRoute<void>(
-            builder: (BuildContext context) => const AllUserProductList(),
-          ),
-        ),
-        icon: Icon(ConstantIcons.instance.getForwardIcon()),
-      ));
       final List<PreferencePageType> items = <PreferencePageType>[
-        PreferencePageType.PROFILE,
+        PreferencePageType.ACCOUNT,
+        PreferencePageType.LISTS,
         PreferencePageType.FOOD,
         PreferencePageType.SETTINGS,
         PreferencePageType.CONTRIBUTE,
         PreferencePageType.FAQ,
         PreferencePageType.CONNECT,
         if (userPreferences.devMode > 0) PreferencePageType.DEV_MODE,
       ];
+
       for (final PreferencePageType type in items) {
-        children.add(getUserPreferences(type).getOnlyHeader());
+        children.add(
+          getUserPreferences(
+            type: type,
+            userPreferences: userPreferences,
+          ).getOnlyHeader(),
+        );
       }
+
       appBarTitle = appLocalizations.myPreferences;
+      addDividers = true;
     } else {
       final AbstractUserPreferences abstractUserPreferences =
-          getUserPreferences(widget.type!);
+          getUserPreferences(
+        type: widget.type!,
+        userPreferences: userPreferences,
+      );
+
       children.addAll(abstractUserPreferences.getContent(withHeader: false));
       appBarTitle = abstractUserPreferences.getTitleString();
+      addDividers = false;
     }
+
     return Scaffold(
       appBar: AppBar(title: Text(appBarTitle)),
-      body: ListView(
-        padding: const EdgeInsets.fromLTRB(0.0, MEDIUM_SPACE, 0.0, 0.0),
-        children: children,
+      body: Scrollbar(
+        child: ListView.builder(
+          padding: const EdgeInsets.only(
+            top: MEDIUM_SPACE,
+          ),
+          // Inject dividers
+          itemCount:
+              addDividers ? ((children.length) * 2) - 1 : children.length,
+          itemBuilder: (BuildContext context, int position) {
+            if (!addDividers) {
+              return children[position];
+            } else if ((position % 2).isOdd) {
+              return const UserPreferencesListItemDivider();
+            } else {
+              return children[(position / 2).round()];
+            }
+          },
+        ),
       ),
     );
   }
+
+  AbstractUserPreferences getUserPreferences({
+    required final PreferencePageType type,
+    required final UserPreferences userPreferences,
+  }) {
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+    final ThemeProvider themeProvider = context.watch<ThemeProvider>();
```

Comment:
I don't think we have to watch as the whole MaterialApp gets rebuild on changes, not 100% sure though

Reply:
It was just a copy/paste, but you're right 👍